tests/integration/relationships/inverse-relationships-test.js

```diff
@@ -640,9 +640,11 @@ testInDebug("Inverse null relationships with models that don't exist throw a nic
 
   let env = setupStore({ user: User });
 
-  assert.throws(() => {
-    run(() => env.store.createRecord('user', { post: {}}));
-  }, /No model was found for 'post'/);
+  run(() => {
```

Comment:
This change seems somewhat odd

Reply:
Indeed, not sure what was going on there. I just moved the runloop back inside the `expectAssertion` and everything passes so I'm going to commit that change.